CMakeLists.txt

```diff
@@ -195,9 +195,6 @@ endif() # TRITON_ENABLE_GCS
 if(${TRITON_ENABLE_S3})
   set(TRITON_DEPENDS ${TRITON_DEPENDS} aws-sdk-cpp)
 endif() # TRITON_ENABLE_S3
-if(${TRITON_ENABLE_AZURE_STORAGE})
-  set(TRITON_DEPENDS ${TRITON_DEPENDS} azure-storage-cpplite)
-endif() # TRITON_ENABLE_AZURE_STORAGE
```

Comment:
Nice to move this into logic core repo completely. I think it would be a good cleanup/followup to move the other core deps strictly into the core repo where it makes sense, and only set the corresponding `TRITON_ENABLE_*` here in server, as you did for azure.